publiccode.yml

```
@@ -8,11 +8,21 @@ categories:
 description:
   en:
     features:
-      - Python Template
-    genericName: python_project
+      - AI
+      - Validation
+      - Transparency
+      - Algorithm
+    genericName: tad
     longDescription: |
-      This a an example python template project.
-    shortDescription: Python Template project
+      TAD is the acronym for Transparency of Algorithmic Decision making. TAD has the goal to make algorithmic systems more transparent; it achieves this by generating standardized reports on the algorithmic system which encompasses both technical aspects in addition to descriptive information about the system and regulatory assessments. For both the system and the model the lifecycle is important and this needs to be taken into account. The definition for an algorithm is derived from the Algoritmeregister.
+
+      On the one hand the TAD project is focused on building a standardized format of reporting on a algorithmic system by developing a Reporting Standard. This Reporting Standard consists out of a System Card which contains Model Cards and Assessment Cards.
+
+      On the other hand the project is focused on filling in the System, Model and Assessment Cards with both performance metrics and technical measurements on fairness and bias of the model, assessments on the system where the specific algorithm resides, and descriptive information about the system. The requirements and instruments are dictated by the Algoritmekader.
```

I assume this is a combination of several sources. It duplicates a few times. I also wonder if it is needed to be verbose about developing the standard, as this indeed part of the TAD project, but not specifically this project. 

```suggestion
TAD is the acronym for Transparency of Algorithmic Decision making. The goal is to make algorithmic systems more transparant. It achieves this by generating standardized reports. These reports consist of both technical aspects as well as descriptive information about the system and regulatory assessments. For both the system and the model the lifecycle is important, so this is incorporated in both the tool as well as the reports. The definition for an algorithm is derived from the Algoritmeregister.
```